Sources/Subprocess/API.swift

```diff
@@ -160,6 +160,11 @@ public func run<Result, Input: InputProtocol, Output: OutputProtocol, Error: Out
 ///     when running the executable.
 ///   - input: The input to send to the executable.
 ///   - error: How to manager executable standard error.
+///   - preferredBufferSize: The preferred size in bytes for the buffer used when reading
```

Comment:
The parameter name (preferredOutputSize) and documentation (preferredBufferSize) disagree on the name.